demo/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Kaltura Player Demo Page" />
+    <title>Download Plugin</title>
+    <link rel="stylesheet" type="text/css" href="./style.css" />
+    <script src="https://unpkg.com/@playkit-js/kaltura-player-js@canary/dist/kaltura-ovp-player.js"></script>
+    <script src="https://unpkg.com/@playkit-js/ui-managers@canary/dist/playkit-ui-managers.js" type="text/javascript"></script>
+    <script src="./playkit-downloads.js" type="text/javascript"></script>
+  </head>
+  <body>
+    <div id="player-placeholder"></div>
+    <script type="text/javascript">
+      const config = {
+        log: {
+          level: 'DEBUG'
+        },
+        targetId: 'player-placeholder',
+        provider: {
+          "env": {
+            "serviceUrl": "https://cdnapisec.kaltura.com/api_v3",
```

Comment:
most of the urls arent used by the plugin, why not just keep the defaults ?

Reply:
it is not working when i remove those lines (this page is just for us to use as test page in dev )